src/acceptor_pool.erl

```diff
@@ -346,19 +388,32 @@ init(Name, Mod, Args, {ok, {#{} = Flags, [#{} = Spec]}}) ->
             Shutdown = maps:get(shutdown, Spec, shutdown_default(Type)),
             Grace = maps:get(grace, Spec, 0),
             Modules = maps:get(modules, Spec, [AMod]),
-            State = #state{name=Name, mod=Mod, args=Args, id=Id, start=Start,
+            State = #state{name=Name, pool_name=PoolName, mod=Mod, args=Args, id=Id, start=Start,
                            restart=Restart, shutdown=Shutdown, grace=Grace,
                            type=Type, modules=Modules, intensity=Intensity,
                            period=Period},
+            _ = make_ets_conn_cache(PoolName),
             {ok, State};
         {error, Reason} ->
             {stop, Reason}
     end;
-init(_, _, _, ignore) ->
+init(_, _, _, _, ignore) ->
     ignore;
-init(_, Mod, _, Other) ->
+init(_, _, Mod, _, Other) ->
     {stop, {bad_return, {Mod, init, Other}}}.
 
+-spec make_ets_conn_cache(atom()) -> atom().
+make_ets_conn_cache(PoolName) ->
```

Comment:
Why is this named/public, can we make it anonymous/protected, or does the pid structure not allow for this?

Reply:
I defaulted to public whilst working out the usage, will change to protected